src/redis-cli.c

```diff
@@ -172,6 +175,7 @@ typedef struct clusterManagerCommand {
     char **argv;
     int flags;
     int replicas;
+    int mode;
```

Comment:
Why don't you use the already existing `int flags`. I don't think it's necessary to create another member in the struct here.

Reply:
done